spinalcordtoolbox/scripts/sct_image.py

```diff
@@ -488,6 +506,14 @@ def multicomponent_merge(fname_list):
     return im_out
 
 
+def read_and_fix_sform(fname):
+    im_in = Image(fname, check_sform=False)
+    im_reaff = im_in.copy()
+    im_reaff.header = im_in.header.copy()
+    im_reaff.header.set_sform(im_reaff.header.get_qform())
+    return im_reaff
```

Comment:
To follow [API code vs application code](https://github.com/neuropoly/spinalcordtoolbox/wiki/Programming%3A-API-code-vs-application-code), it could be better to put this functionality into the API, via `spinalcordtoolbox/image.py`.

```python
# Inside spinalcordtoolbox/image.py
class Image(object):
    # [...]
    def set_sform_to_qform(self):
        """Use this (or set_qform_to_sform) when matching matrices are required."""
        self.header.set_sform(self.header.get_qform())
 
    def set_qform_to_sform(self):
        """Use this or (set_sform_to_qform) when matching matrices are required."""
        self.header.set_qform(self.header.get_sform())
```